app/tests/spec/views/sign_up.js

```diff
@@ -528,10 +528,6 @@ define(function (require, exports, module) {
         view.showValidationErrors();
 
         assert.isTrue(view.showValidationError.called);
-
-        var err = AuthErrors.toError('DIFFERENT_EMAIL_REQUIRED_FIREFOX_DOMAIN');
```

Comment:
Yep. It was purely out of simplicity though, if it's a problem I'm happy to do something about it.